x-pack/plugin/eql/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/eql/10_basic.yml

```diff
@@ -83,6 +83,20 @@ setup:
   - match: {hits.sequences.1.events.0._id: "2"}
   - match: {hits.sequences.1.events.1._id: "3"}
 
+---
+"Execute EQL sequence by default ignores unavailable indices"
+  - do:
+      eql.search:
+        index: eql_test,non_existing
```

Comment:
@matriv the integration test against non-existing indices

Reply:
I meant to query against only unavailable indices, just for completeness.